var/spack/repos/builtin/packages/diy/diy-mr82-add-smoke-test.patch

```diff
@@ -0,0 +1,859 @@
+From 1d85dd5205b9f0035840e1840a49ea7028618d16 Mon Sep 17 00:00:00 2001
```

Comment:
I would prefer not to commit this patch into Spack as it is quite large. Could you grab this patch as a URL to the diff or ideally just enable the examples in newer versions that have the smoke tests in them.

Reply:
Ill go ahead with Option 1 since option 2 is not possible, the current examples at diy are not "portable" the have dependency to headers out of their dir and their cmakelists is not standalone.

Reply:
@kwryankrattiger resolved in the last push. Btw no hurry for this, lets wait till this patch gets upstream :)